### PR TITLE
feat: CLIN-2165 - coverage_by_gene_centric

### PR DESCRIPTION
**Pull request by creativeyann17**
- [x] Enriched CorevageByGene filter refseq by `type=gene` (remove duplicated)
- [x] Enriched CorevageByGene add `hash` field (required for front-end table)
- [x] Prepare CorevageByGene Centric
- [x] CorevageByGene template ES
- [x] EtlConfiguration
- [x] Add Tests shouldNotBeEmpty for `corverage_by_gene` and `corverage_by_gene_centric`
- [x] TUs
- [x] Test QA

### CODE REVIEW

File: src/main/resources/templates/coverage_by_gene_centric_template.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          "type": "long"
+        },
+        "end": {
+          "type": "long"
+        },
+        "service_request_id": {
+          "type": "keyword"
+        },
+        "ensembl_gene_id": {
+          "type": "keyword"
```
Comment (jecos) [inline src/main/resources/templates/coverage_by_gene_centric_template.json:72]: index false

File: src/main/resources/templates/coverage_by_gene_centric_template.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+          "type": "float"
+        },
+        "coverage500": {
+          "type": "float"
+        },
+        "coverage1000": {
+          "type": "float"
+        },
+        "aliquot_id": {
+          "type": "keyword"
```
Comment (jecos) [inline src/main/resources/templates/coverage_by_gene_centric_template.json:54]: index false

File: src/main/resources/templates/coverage_by_gene_centric_template.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        },
+        "batch_id": {
+          "type": "keyword"
```
Comment (jecos) [inline src/main/resources/templates/coverage_by_gene_centric_template.json:57]: index false

**Review by jecos — CHANGES_REQUESTED** [review]

File: src/main/scala/bio/ferlab/clin/etl/es/Indexer.scala [hunk 4]
```diff
@@ -54,6 +54,7 @@ object Indexer extends App {
     case "variant_centric" => conf.getDataset("es_index_variant_centric")
     case "cnv_centric" => conf.getDataset("es_index_cnv_centric")
     case "variant_suggestions" => conf.getDataset("es_index_variant_suggestions")
+    case "coverage_by_gene_centric" => conf.getDataset("es_index_coverage_by_gene_centric")
```
[the remark this answers is not in the document]
Comment (creativeyann17) [reply src/main/scala/bio/ferlab/clin/etl/es/Indexer.scala:57]: We have to use the one from Prepare `es_index_coverage_by_gene_centric` which contains the release number as suffix.

**Pull request merged**